Jamfile

```diff
@@ -0,0 +1,13 @@
+#  Boost.DateTime Library Jamfile
+#
+#  Copyright (c) 2018 James E. King III
+#
+#  Use, modification, and distribution are subject to the
+#  Boost Software License, Version 1.0. (See accompanying file
+#  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+project libs/date_time ;
```

Comment:
Not sure that one should go out of his way to add project names when they aren't needed; seems to me that this brings no benefit and can only cause problems.

Reply:
Okay, I can remove it - should it also be removed from libs/date_time/test/Jamfile as well?  Only the build/ one should have a project by name?  (Also curious why date_time has a library at all... it doesn't look like it really needs one for the scant few items in src/)

Still working out the MinGW issues; got cygwin building clean and fixed a y2038 issue so far in my build investigation.